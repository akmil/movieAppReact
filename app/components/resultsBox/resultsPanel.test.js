import React from 'react';
// import renderer from 'react-test-renderer';
import ResultsBox , {ResultsBodyItem, ResultsBody} from "./resultsBox";
import { shallow } from 'enzyme'

describe('resultsBox', function () {
    it('should render without errors ResultsBodyItem', () => {
        const wrapper = shallow(<ResultsBodyItem/>);
        expect(wrapper).toMatchSnapshot();
    });

    it('should render without errors ResultsBody', () => {
        const wrapper = shallow(<ResultsBody/>);
        expect(wrapper).toMatchSnapshot();
    });
    it('should render without errors ResultsBody', () => {
        const wrapper = shallow(<ResultsBox/>);
        expect(wrapper).toMatchSnapshot();
    });
});
