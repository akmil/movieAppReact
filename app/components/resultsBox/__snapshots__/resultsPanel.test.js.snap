// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resultsBox should render without errors ResultsBody 1`] = `
<div
  className="results-list-box"
>
  <ul
    className="results-list"
  >
    <ResultsBodyItem
      genre="Drama2"
      key="0"
      onClick={[Function]}
      releaseDate="1299"
      src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      title="Movie title 4"
    />
    <ResultsBodyItem
      genre="Drama2"
      key="1"
      onClick={[Function]}
      releaseDate="1395"
      src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      title="Movie title 5"
    />
    <ResultsBodyItem
      genre="Drama"
      releaseDate="1995"
      src="https://images.pexels.com/photos/460307/pexels-photo-460307.jpeg?auto=compress&cs=tinysrgb&h=350"
      title="Movie title 1"
    />
    <ResultsBodyItem
      genre="Comedy"
      releaseDate="1998"
      src="https://images.pexels.com/photos/257092/pexels-photo-257092.jpeg?auto=compress&cs=tinysrgb&h=350"
      title="Movie title 2"
    />
    <ResultsBodyItem
      genre="Drama"
      releaseDate="1991"
      src="https://images.pexels.com/photos/374633/pexels-photo-374633.jpeg?auto=compress&cs=tinysrgb&h=350"
      title="Movie title 3"
    />
  </ul>
</div>
`;

exports[`resultsBox should render without errors ResultsBody 2`] = `
<React.Fragment>
  <ResultsBody />
</React.Fragment>
`;

exports[`resultsBox should render without errors ResultsBodyItem 1`] = `
<li>
  <img />
  <div
    className="col-50"
  >
    <p
      className="title"
    />
    <p
      className="genre"
    />
  </div>
  <div
    className="col-50 text-align-right"
  >
    <span
      className="release-date"
    />
  </div>
</li>
`;
